src/homework/day8/Doubles.java

```diff
@@ -5,8 +5,8 @@
 
 public class Doubles {
     public static void main(String[] args) {
-        Double myDoub[] = {33.42, 34.3, 0.79, 2.3426, 6.8, 13.24, 5.5, 769.9};
-        List<Double> doubles = Arrays.asList(myDoub);
+        Double myDoubles[] = {33.42, 34.3, 0.79, 2.3426, 6.8, 13.24, 5.5, 769.9};
+        List<Double> doubles = Arrays.asList(myDoubles);
 
```

Comment:
It's not necessary to create array myDoubt[], you can fill it in an  Arrays.asList, like Arrays.asList(33.42, 34.3);